source/stylesheets/base/_layout.scss

```diff
@@ -0,0 +1,64 @@
+/* This is the base.erb layout:
+ * #page {}
+ *   #header {}
+ *   #main {}
+ *     #content-wrapper {}
+ *       #content {}
+ *     #sidebar-wrapper {}
+ *       #sidebar {}
+ *   #footer {}
+ */
+
+*, :before, :after {
+  box-sizing: border-box;
```

Comment:
So that's present in both [Bootstrap](https://github.com/twbs/bootstrap/blob/ebad168f4296e83e36ee889667ba8907c1a93677/scss/_reboot.scss#L16-L20) & [TailwindCSS](https://github.com/tailwindlabs/tailwindcss/blob/2b0b306a77cc0f34cd25054a0d661a18278786bc/src/plugins/css/preflight.css#L99), I added it because I was a little thrown off by it not being present. 

[Bootstrap has documented their rational](https://getbootstrap.com/docs/4.1/getting-started/introduction/#box-sizing), which I do agree with. I think perhaps I need to setup a more explicit for this kind of change?